evm/forge-test/CircleRelayerGovernance.t.sol

```diff
@@ -782,4 +780,59 @@ contract CircleRelayerGovernanceTest is Test, ForgeHelpers {
 
         vm.stopPrank();
     }
+
```

Comment:
Can we add a couple of tests here? 
1) Only-owner test that confirms that only the owner can toggle the `pause` functionality. 
2) Test that shows that the contract will send outbound transfers after disabling the pausing mechanism.